### DISCUSSION TITLE
faster-whisper don't support turbo model, please update

### DISCUSSION THREAD

**Discussion by kitaev-chen** [open-webui/open-webui#7116] [General]
# Bug Report

When choosing Whisper (Local) as Speech-to-Text Engine and change the STT Model from **base** to **turbo**, there is an error show in docker log:

```
File "/usr/local/lib/python3.11/site-packages/faster_whisper/utils.py", line 78, in download_model
    raise ValueError(
ValueError: Invalid model size 'turbo', expected one of: tiny.en, tiny, base.en, base, small.en, small, medium.en, medium, large-v1, large-v2, large-v3, large, distil-large-v2, distil-medium.en, distil-small.en, distil-large-v3
```

It looks like the faster_whisper inside open-webui doesn't support [faster-large-v3-turbo](https://huggingface.co/deepdml/faster-whisper-large-v3-turbo-ct2) model. When checking faster_whisper [model list](https://github.com/SYSTRAN/faster-whisper/blob/master/faster_whisper/utils.py), it already supports this turbo model. Please see the [benchmark](https://github.com/SYSTRAN/faster-whisper/issues/1030).

---

## Installation Method
Install from Docker latest image

## Environment
- **Open WebUI Version:** [v0.4.1]
- **Ollama (if applicable):** [v0.4.2]
- **Operating System:** [Windows 11]

**Comment by 3dfxgamer** [comment]
Hi! did you find a fix for this, i`m trying the same.

**Comment by hongbo-miao** [comment]
I think this need wait upstream faster-whisper to support first which tracks at https://github.com/SYSTRAN/faster-whisper/issues/1030

**Comment by MrGenia** [comment]
Following up on this thread, I've created a discussion with a proposed solution for adding turbo model support to Open WebUI: [#9725](https://github.com/open-webui/open-webui/discussions/9725)

**Reply by MrGenia** [reply]
Commit [3e14529](https://github.com/open-webui/open-webui/commit/3e1452902fd5a7eb85a6cab9185fbd5577e96893) was merged into the dev branch
